### PR TITLE
Test cases added for type2 ODE's in issue 19763

### PR DESCRIPTION
**Pull request by kunalsingh2002**
#### References to other Issues or PRs

Implements test cases from PR #19888 , followup of #20673

#### Brief description of what is fixed or changed

Added test cases for type 2 ODE's

#### Release Notes

* solvers
  * added 14 test cases

### CODE REVIEW

**Comment by sidhu1012** [comment]
No need of release notes in this case.

**Comment by oscarbenjamin** [comment]
You should mention that this is a followup of #20673 

How exactly did you choose these test cases?

Were any of these included in #20672?

**Comment by kunalsingh2002** [comment]
> You should mention that this is a followup of #20673
> 
> How exactly did you choose these test cases?
> 
> Were any of these included in #20672?

No sir none of these test cases are in #20672 , these are test cases for type 2 and #20672 is for type 3
I used PR #19888 and included these tests and removed the error just like @suryam35 did in #20672

**Comment by kunalsingh2002** [comment]
> You should mention that this is a followup of #20673

Updated sir

File: sympy/solvers/ode/tests/test_systems.py [hunk 1]
```diff
@@ -1150,6 +1150,119 @@ def test_sysode_linear_neq_order1_type2():
...
+    assert checksysodesol(eq25 , sol25) == (True , [0,0])
+
+    eq26 = [Eq(Derivative(f(t), t), 2*f(t) - g(t) + k1),
+            Eq(Derivative(g(t), t), 4*f(t) - 2*g(t) + 2*k1)]
+    sol26 = [Eq(f(t), C1 + 2*C2 + t*(2*C1 + k1)),
+            Eq(g(t), 4*C2 + t*(4*C1 + 2*k1))]
+    assert dsolve(eq26) == sol26
+    assert checksysodesol(eq26 , sol26) == (True , [0,0])
+
+
```
Comment (oscarbenjamin) [inline sympy/solvers/ode/tests/test_systems.py:1266]: There should normally be only 2 blank lines between top-level functions although I see that's already not entirely consistent in this file

**Comment by oscarbenjamin** [comment]
Okay, looks good. Thanks!

**Comment by oscarbenjamin** [comment]
> none of these test cases are in #20672 , these are test cases for type 2 and #20672 is for type 3

Does that mean that all the tests are added now?

**Comment by suryam35** [comment]
> > none of these test cases are in #20672 , these are test cases for type 2 and #20672 is for type 3
> 
> Does that mean that all the tests are added now?

Yes , all the test cases have been added now

**Comment by oscarbenjamin** [comment]
Great. Thanks!

**Comment by sidhu1012** [comment]
> No need of release notes in this case.

Adding test cases shouldn't have release notes.

**Comment by naveensaigit** [comment]
> Adding test cases shouldn't have release notes.

Yes, but now that its merged, it can be removed when the next release is done.

**Comment by oscarbenjamin** [comment]
Good point. There shouldn't be a release note.

It can be removed now just by editing the release notes page:
https://github.com/sympy/sympy/wiki/Release-Notes-for-1.8

**Comment by kunalsingh2002** [comment]
> Good point. There shouldn't be a release note.
> 
> It can be removed now just by editing the release notes page:
> https://github.com/sympy/sympy/wiki/Release-Notes-for-1.8

Updated the release notes

**Pull request merged**